Error: No static data to generate the blank data. please make sure the network connection is reachable to Unicode.org

I got this message 
Error: No static data to generate the blank data. please make sure the network connection is reachable to Unicode.org
make[2]: Leaving directory '/tmp/compile/fontconfig-2.12.6/fc-blanks'
make[2]: *** [Makefile:595: fcblanks.h] Error 1
make[1]: *** [Makefile:581: all-recursive] Error 1
make[1]: Leaving directory '/tmp/compile/fontconfig-2.12.6'
make: *** [Makefile:465: all] Error 2
The command '/bin/sh -c ./configure --prefix=${C} --host=${CROSS_ARCH} --build=${BUILD_ARCH} --enable-static --disable-shared &&     make &&     make install' returned a non-zero code: 2
+ exit 1

yes, I have
it's already in step 211 
and it downlaoded many things already

Step 211/228 : RUN ./configure --prefix=${C} --host=${CROSS_ARCH} --build=${BUILD_ARCH} --enable-static --disable-shared &&     make &&     make install
 ---> Running in 2cf16021a9b3
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for x86_64-w64-mingw32-strip... x86_64-w64-mingw32-strip
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... no
checking for mawk... mawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking for x86_64-w64-mingw32-gcc... x86_64-w64-mingw32-gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.exe
checking for suffix of executables... .exe
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether x86_64-w64-mingw32-gcc accepts -g... yes
checking for x86_64-w64-mingw32-gcc option to accept ISO C89... none needed
checking whether x86_64-w64-mingw32-gcc understands -c and -o together... yes
checking for style of include used by make... GNU
checking dependency style of x86_64-w64-mingw32-gcc... gcc3
checking how to run the C preprocessor... x86_64-w64-mingw32-gcc -E
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... 64
checking whether ln -s works... yes
checking whether make sets $(MAKE)... (cached) yes
checking for x86_64-w64-mingw32-pkg-config... /tmp/compile/bin/x86_64-w64-mingw32-pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for python... /usr/bin/python
checking for python version... 2.7
checking for python platform... linux2
checking for python script directory... ${prefix}/lib/python2.7/site-packages
checking for python extension module directory... ${exec_prefix}/lib/python2.7/site-packages
checking for RM macro... rm -f
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-w64-mingw32
checking how to print strings... printf
checking for a sed that does not truncate output... /usr/bin/sed
checking for fgrep... /usr/bin/grep -F
checking for ld used by x86_64-w64-mingw32-gcc... /usr/bin/x86_64-w64-mingw32-ld
checking if the linker (/usr/bin/x86_64-w64-mingw32-ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/x86_64-w64-mingw32-nm -B
checking the name lister (/usr/bin/x86_64-w64-mingw32-nm -B) interface... BSD nm
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-pc-linux-gnu file names to x86_64-w64-mingw32 format... func_convert_file_nix_to_w32
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/x86_64-w64-mingw32-ld option to reload object files... -r
checking for x86_64-w64-mingw32-objdump... x86_64-w64-mingw32-objdump
checking how to recognize dependent libraries... file_magic ^x86 archive import|^x86 DLL
checking for x86_64-w64-mingw32-dlltool... x86_64-w64-mingw32-dlltool
checking how to associate runtime and link libraries... func_cygming_dll_for_implib
checking for x86_64-w64-mingw32-ar... x86_64-w64-mingw32-ar
checking for archiver @FILE support... @
checking for x86_64-w64-mingw32-strip... (cached) x86_64-w64-mingw32-strip
checking for x86_64-w64-mingw32-ranlib... x86_64-w64-mingw32-ranlib
checking command to parse /usr/bin/x86_64-w64-mingw32-nm -B output from x86_64-w64-mingw32-gcc object... ok
checking for sysroot... no
checking for a working dd... /usr/bin/dd
checking how to truncate binary pipes... /usr/bin/dd bs=4096 count=1
checking for x86_64-w64-mingw32-mt... no
checking for mt... no
checking if : is a manifest tool... no
checking for dlfcn.h... no
checking for x86_64-w64-mingw32-as... x86_64-w64-mingw32-as
checking for x86_64-w64-mingw32-dlltool... (cached) x86_64-w64-mingw32-dlltool
checking for x86_64-w64-mingw32-objdump... (cached) x86_64-w64-mingw32-objdump
checking for objdir... .libs
checking if x86_64-w64-mingw32-gcc supports -fno-rtti -fno-exceptions... no
checking for x86_64-w64-mingw32-gcc option to produce PIC... -DDLL_EXPORT -DPIC
checking if x86_64-w64-mingw32-gcc PIC flag -DDLL_EXPORT -DPIC works... yes
checking if x86_64-w64-mingw32-gcc static flag -static works... yes
checking if x86_64-w64-mingw32-gcc supports -c -o file.o... yes
checking if x86_64-w64-mingw32-gcc supports -c -o file.o... (cached) yes
checking whether the x86_64-w64-mingw32-gcc linker (/usr/bin/x86_64-w64-mingw32-ld) supports shared libraries... yes
checking dynamic linker characteristics... Win32 ld.exe
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... no
checking whether to build static libraries... yes
checking for lib.exe... no
checking whether __SUNPRO_C is declared... no
checking for build system executable suffix... no
checking for dirent.h that defines DIR... yes
checking for library containing opendir... none required
checking for ANSI C header files... (cached) yes
checking fcntl.h usability... yes
checking fcntl.h presence... yes
checking for fcntl.h... yes
checking for stdlib.h... (cached) yes
checking for string.h... (cached) yes
checking for unistd.h... (cached) yes
checking sys/statvfs.h usability... no
checking sys/statvfs.h presence... no
checking for sys/statvfs.h... no
checking sys/vfs.h usability... no
checking sys/vfs.h presence... no
checking for sys/vfs.h... no
checking sys/statfs.h usability... no
checking sys/statfs.h presence... no
checking for sys/statfs.h... no
checking sys/param.h usability... yes
checking sys/param.h presence... yes
checking for sys/param.h... yes
checking sys/mount.h usability... no
checking sys/mount.h presence... no
checking for sys/mount.h... no
checking for stdint types... stdint.h (shortcircuit)
make use of stdint.h in src/fcstdint.h (assuming C99 compatible system)
checking for an ANSI C-conforming const... yes
checking for inline... inline
checking for flexible array members... yes
checking for pid_t... yes
checking for vprintf... yes
checking for _doprnt... no
checking for stdlib.h... (cached) yes
checking for unistd.h... (cached) yes
checking for sys/param.h... (cached) yes
checking for getpagesize... yes
checking for working mmap... no
checking for link... no
checking for mkstemp... yes
checking for mkostemp... no
checking for _mktemp_s... yes
checking for mkdtemp... no
checking for getopt... yes
checking for getopt_long... yes
checking for getprogname... no
checking for getexecname... no
checking for rand... yes
checking for random... no
checking for lrand48... no
checking for random_r... no
checking for rand_r... no
checking for readlink... no
checking for fstatvfs... no
checking for fstatfs... no
checking for lstat... no
checking for posix_fadvise in fcntl.h... no
checking for struct stat.st_mtim... no
checking for struct dirent.d_type... no
checking The type of len parameter of gperf hash/lookup function... size_t
checking for FREETYPE... yes
checking for FT_Get_Next_Char... yes
checking for FT_Get_BDF_Property... yes
checking for FT_Get_PS_Font_Info... yes
checking for FT_Has_PS_Glyph_Names... yes
checking for FT_Get_X11_Font_Format... yes
checking for FT_Select_Size... yes
checking for FT_Bitmap_Size.y_ppem... yes
checking for TT_OS2.usLowerOpticalPointSize... yes
checking for TT_OS2.usUpperOpticalPointSize... yes
checking for EXPAT... yes
checking expat.h usability... yes
checking expat.h presence... yes
checking for expat.h... yes
checking for XML_SetDoctypeDeclHandler... yes
checking for Intel atomic primitives... true
checking for Solaris atomic operations... false
checking for docbook2html... no
checking whether byte ordering is bigendian... no
checking size of void *... 8
checking alignment of double... 8
checking alignment of void *... 8
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating fontconfig/Makefile
config.status: creating fc-lang/Makefile
config.status: creating fc-glyphname/Makefile
config.status: creating fc-blanks/Makefile
config.status: creating fc-case/Makefile
config.status: creating src/Makefile
config.status: creating conf.d/Makefile
config.status: creating fc-cache/Makefile
config.status: creating fc-cat/Makefile
config.status: creating fc-list/Makefile
config.status: creating fc-match/Makefile
config.status: creating fc-pattern/Makefile
config.status: creating fc-query/Makefile
config.status: creating fc-scan/Makefile
config.status: creating fc-validate/Makefile
config.status: creating doc/Makefile
config.status: creating doc/version.sgml
config.status: creating test/Makefile
config.status: creating fontconfig.spec
config.status: creating fontconfig.pc
config.status: creating fontconfig-zip
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands
config.status: executing src/fcstdint.h commands
config.status: creating src/fcstdint.h : _FONTCONFIG_SRC_FCSTDINT_H
make  all-recursive
make[1]: Entering directory '/tmp/compile/fontconfig-2.12.6'
Making all in fontconfig
make[2]: Entering directory '/tmp/compile/fontconfig-2.12.6/fontconfig'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/tmp/compile/fontconfig-2.12.6/fontconfig'
Making all in fc-blanks
make[2]: Entering directory '/tmp/compile/fontconfig-2.12.6/fc-blanks'
  GEN      fcblanks.h
Error: No static data to generate the blank data. please make sure the network connection is reachable to Unicode.org
make[2]: *** [Makefile:595: fcblanks.h] Error 1
make[2]: Leaving directory '/tmp/compile/fontconfig-2.12.6/fc-blanks'
make[1]: Leaving directory '/tmp/compile/fontconfig-2.12.6'
make[1]: *** [Makefile:581: all-recursive] Error 1
make: *** [Makefile:465: all] Error 2
The command '/bin/sh -c ./configure --prefix=${C} --host=${CROSS_ARCH} --build=${BUILD_ARCH} --enable-static --disable-shared &&     make &&     make install' returned a non-zero code: 2
+ exit 1

And can you reach the Unicode.org server?

AhmadQG <notifications@github.com> schrieb am Fr., 5. Juni 2020, 11:51:

> yes, I have
> it's already in step 211
> and it downlaoded many things already
>
> Step 211/228 : RUN ./configure --prefix=${C} --host=${CROSS_ARCH}
> --build=${BUILD_ARCH} --enable-static --disable-shared && make && make
> install
> ---> Running in 2cf16021a9b3
> checking for a BSD-compatible install... /usr/bin/install -c
> checking whether build environment is sane... yes
> checking for x86_64-w64-mingw32-strip... x86_64-w64-mingw32-strip
> checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
> checking for gawk... no
> checking for mawk... mawk
> checking whether make sets $(MAKE)... yes
> checking whether make supports nested variables... yes
> checking whether make supports nested variables... (cached) yes
> checking for x86_64-w64-mingw32-gcc... x86_64-w64-mingw32-gcc
> checking whether the C compiler works... yes
> checking for C compiler default output file name... a.exe
> checking for suffix of executables... .exe
> checking whether we are cross compiling... yes
> checking for suffix of object files... o
> checking whether we are using the GNU C compiler... yes
> checking whether x86_64-w64-mingw32-gcc accepts -g... yes
> checking for x86_64-w64-mingw32-gcc option to accept ISO C89... none needed
> checking whether x86_64-w64-mingw32-gcc understands -c and -o together...
> yes
> checking for style of include used by make... GNU
> checking dependency style of x86_64-w64-mingw32-gcc... gcc3
> checking how to run the C preprocessor... x86_64-w64-mingw32-gcc -E
> checking for grep that handles long lines and -e... /usr/bin/grep
> checking for egrep... /usr/bin/grep -E
> checking for ANSI C header files... yes
> checking for sys/types.h... yes
> checking for sys/stat.h... yes
> checking for stdlib.h... yes
> checking for string.h... yes
> checking for memory.h... yes
> checking for strings.h... yes
> checking for inttypes.h... yes
> checking for stdint.h... yes
> checking for unistd.h... yes
> checking minix/config.h usability... no
> checking minix/config.h presence... no
> checking for minix/config.h... no
> checking whether it is safe to define *EXTENSIONS*... yes
> checking for special C compiler options needed for large files... no
> checking for _FILE_OFFSET_BITS value needed for large files... 64
> checking whether ln -s works... yes
> checking whether make sets $(MAKE)... (cached) yes
> checking for x86_64-w64-mingw32-pkg-config...
> /tmp/compile/bin/x86_64-w64-mingw32-pkg-config
> checking pkg-config is at least version 0.9.0... yes
> checking for python... /usr/bin/python
> checking for python version... 2.7
> checking for python platform... linux2
> checking for python script directory...
> ${prefix}/lib/python2.7/site-packages
> checking for python extension module directory...
> ${exec_prefix}/lib/python2.7/site-packages
> checking for RM macro... rm -f
> checking build system type... x86_64-pc-linux-gnu
> checking host system type... x86_64-w64-mingw32
> checking how to print strings... printf
> checking for a sed that does not truncate output... /usr/bin/sed
> checking for fgrep... /usr/bin/grep -F
> checking for ld used by x86_64-w64-mingw32-gcc...
> /usr/bin/x86_64-w64-mingw32-ld
> checking if the linker (/usr/bin/x86_64-w64-mingw32-ld) is GNU ld... yes
> checking for BSD- or MS-compatible name lister (nm)...
> /usr/bin/x86_64-w64-mingw32-nm -B
> checking the name lister (/usr/bin/x86_64-w64-mingw32-nm -B) interface...
> BSD nm
> checking the maximum length of command line arguments... 1572864
> checking how to convert x86_64-pc-linux-gnu file names to
> x86_64-w64-mingw32 format... func_convert_file_nix_to_w32
> checking how to convert x86_64-pc-linux-gnu file names to toolchain
> format... func_convert_file_noop
> checking for /usr/bin/x86_64-w64-mingw32-ld option to reload object
> files... -r
> checking for x86_64-w64-mingw32-objdump... x86_64-w64-mingw32-objdump
> checking how to recognize dependent libraries... file_magic ^x86 archive
> import|^x86 DLL
> checking for x86_64-w64-mingw32-dlltool... x86_64-w64-mingw32-dlltool
> checking how to associate runtime and link libraries...
> func_cygming_dll_for_implib
> checking for x86_64-w64-mingw32-ar... x86_64-w64-mingw32-ar
> checking for archiver @file <https://github.com/file> support... @
> checking for x86_64-w64-mingw32-strip... (cached) x86_64-w64-mingw32-strip
> checking for x86_64-w64-mingw32-ranlib... x86_64-w64-mingw32-ranlib
> checking command to parse /usr/bin/x86_64-w64-mingw32-nm -B output from
> x86_64-w64-mingw32-gcc object... ok
> checking for sysroot... no
> checking for a working dd... /usr/bin/dd
> checking how to truncate binary pipes... /usr/bin/dd bs=4096 count=1
> checking for x86_64-w64-mingw32-mt... no
> checking for mt... no
> checking if : is a manifest tool... no
> checking for dlfcn.h... no
> checking for x86_64-w64-mingw32-as... x86_64-w64-mingw32-as
> checking for x86_64-w64-mingw32-dlltool... (cached)
> x86_64-w64-mingw32-dlltool
> checking for x86_64-w64-mingw32-objdump... (cached)
> x86_64-w64-mingw32-objdump
> checking for objdir... .libs
> checking if x86_64-w64-mingw32-gcc supports -fno-rtti -fno-exceptions... no
> checking for x86_64-w64-mingw32-gcc option to produce PIC... -DDLL_EXPORT
> -DPIC
> checking if x86_64-w64-mingw32-gcc PIC flag -DDLL_EXPORT -DPIC works... yes
> checking if x86_64-w64-mingw32-gcc static flag -static works... yes
> checking if x86_64-w64-mingw32-gcc supports -c -o file.o... yes
> checking if x86_64-w64-mingw32-gcc supports -c -o file.o... (cached) yes
> checking whether the x86_64-w64-mingw32-gcc linker
> (/usr/bin/x86_64-w64-mingw32-ld) supports shared libraries... yes
> checking dynamic linker characteristics... Win32 ld.exe
> checking how to hardcode library paths into programs... immediate
> checking whether stripping libraries is possible... yes
> checking if libtool supports shared libraries... yes
> checking whether to build shared libraries... no
> checking whether to build static libraries... yes
> checking for lib.exe... no
> checking whether __SUNPRO_C is declared... no
> checking for build system executable suffix... no
> checking for dirent.h that defines DIR... yes
> checking for library containing opendir... none required
> checking for ANSI C header files... (cached) yes
> checking fcntl.h usability... yes
> checking fcntl.h presence... yes
> checking for fcntl.h... yes
> checking for stdlib.h... (cached) yes
> checking for string.h... (cached) yes
> checking for unistd.h... (cached) yes
> checking sys/statvfs.h usability... no
> checking sys/statvfs.h presence... no
> checking for sys/statvfs.h... no
> checking sys/vfs.h usability... no
> checking sys/vfs.h presence... no
> checking for sys/vfs.h... no
> checking sys/statfs.h usability... no
> checking sys/statfs.h presence... no
> checking for sys/statfs.h... no
> checking sys/param.h usability... yes
> checking sys/param.h presence... yes
> checking for sys/param.h... yes
> checking sys/mount.h usability... no
> checking sys/mount.h presence... no
> checking for sys/mount.h... no
> checking for stdint types... stdint.h (shortcircuit)
> make use of stdint.h in src/fcstdint.h (assuming C99 compatible system)
> checking for an ANSI C-conforming const... yes
> checking for inline... inline
> checking for flexible array members... yes
> checking for pid_t... yes
> checking for vprintf... yes
> checking for _doprnt... no
> checking for stdlib.h... (cached) yes
> checking for unistd.h... (cached) yes
> checking for sys/param.h... (cached) yes
> checking for getpagesize... yes
> checking for working mmap... no
> checking for link... no
> checking for mkstemp... yes
> checking for mkostemp... no
> checking for _mktemp_s... yes
> checking for mkdtemp... no
> checking for getopt... yes
> checking for getopt_long... yes
> checking for getprogname... no
> checking for getexecname... no
> checking for rand... yes
> checking for random... no
> checking for lrand48... no
> checking for random_r... no
> checking for rand_r... no
> checking for readlink... no
> checking for fstatvfs... no
> checking for fstatfs... no
> checking for lstat... no
> checking for posix_fadvise in fcntl.h... no
> checking for struct stat.st_mtim... no
> checking for struct dirent.d_type... no
> checking The type of len parameter of gperf hash/lookup function... size_t
> checking for FREETYPE... yes
> checking for FT_Get_Next_Char... yes
> checking for FT_Get_BDF_Property... yes
> checking for FT_Get_PS_Font_Info... yes
> checking for FT_Has_PS_Glyph_Names... yes
> checking for FT_Get_X11_Font_Format... yes
> checking for FT_Select_Size... yes
> checking for FT_Bitmap_Size.y_ppem... yes
> checking for TT_OS2.usLowerOpticalPointSize... yes
> checking for TT_OS2.usUpperOpticalPointSize... yes
> checking for EXPAT... yes
> checking expat.h usability... yes
> checking expat.h presence... yes
> checking for expat.h... yes
> checking for XML_SetDoctypeDeclHandler... yes
> checking for Intel atomic primitives... true
> checking for Solaris atomic operations... false
> checking for docbook2html... no
> checking whether byte ordering is bigendian... no
> checking size of void *... 8
> checking alignment of double... 8
> checking alignment of void *... 8
> checking that generated files are newer than configure... done
> configure: creating ./config.status
> config.status: creating Makefile
> config.status: creating fontconfig/Makefile
> config.status: creating fc-lang/Makefile
> config.status: creating fc-glyphname/Makefile
> config.status: creating fc-blanks/Makefile
> config.status: creating fc-case/Makefile
> config.status: creating src/Makefile
> config.status: creating conf.d/Makefile
> config.status: creating fc-cache/Makefile
> config.status: creating fc-cat/Makefile
> config.status: creating fc-list/Makefile
> config.status: creating fc-match/Makefile
> config.status: creating fc-pattern/Makefile
> config.status: creating fc-query/Makefile
> config.status: creating fc-scan/Makefile
> config.status: creating fc-validate/Makefile
> config.status: creating doc/Makefile
> config.status: creating doc/version.sgml
> config.status: creating test/Makefile
> config.status: creating fontconfig.spec
> config.status: creating fontconfig.pc
> config.status: creating fontconfig-zip
> config.status: creating config.h
> config.status: executing depfiles commands
> config.status: executing libtool commands
> config.status: executing src/fcstdint.h commands
> config.status: creating src/fcstdint.h : _FONTCONFIG_SRC_FCSTDINT_H
> make all-recursive
> make[1]: Entering directory '/tmp/compile/fontconfig-2.12.6'
> Making all in fontconfig
> make[2]: Entering directory '/tmp/compile/fontconfig-2.12.6/fontconfig'
> make[2]: Nothing to be done for 'all'.
> make[2]: Leaving directory '/tmp/compile/fontconfig-2.12.6/fontconfig'
> Making all in fc-blanks
> make[2]: Entering directory '/tmp/compile/fontconfig-2.12.6/fc-blanks'
> GEN fcblanks.h
> Error: No static data to generate the blank data. please make sure the
> network connection is reachable to Unicode.org
> make[2]: *** [Makefile:595: fcblanks.h] Error 1
> make[2]: Leaving directory '/tmp/compile/fontconfig-2.12.6/fc-blanks'
> make[1]: Leaving directory '/tmp/compile/fontconfig-2.12.6'
> make[1]: *** [Makefile:581: all-recursive] Error 1
> make: *** [Makefile:465: all] Error 2
> The command '/bin/sh -c ./configure --prefix=${C} --host=${CROSS_ARCH}
> --build=${BUILD_ARCH} --enable-static --disable-shared && make && make
> install' returned a non-zero code: 2
>
>    - exit 1
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/wincentbalin/compile-static-audiowaveform/issues/3#issuecomment-639377236>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABYCDME5JTNAVEYGGOUES3RVC52VANCNFSM4NTJSFQA>
> .
>

--- a/Dockerfile
+++ b/Dockerfile
@@ -43,8 +43,7 @@ ENV LIQ_VERSION 2.11.9
 ENV FREETYPE2_VERSION 2.10.1
 ENV EXPAT_VERSION 2.2.8
 ENV EXPAT_VERSION_MANGLED 2_2_8
-# Later versions of fontconfig do not supply pre-configured configure script
-ENV FONTCONFIG_VERSION 2.12.6
+ENV FONTCONFIG_VERSION 2.13.92
 ENV LIBGD_VERSION 2.2.5
 ENV BOOST_VERSION 1.65.1
 ENV BOOST_VERSION_MANGLED 1_65_1
@@ -220,8 +219,8 @@ RUN ./configure --prefix=${C} --host=${CROSS_ARCH} --build=${BUILD_ARCH} --enabl
 WORKDIR ..
 
 # Download and compile fontconfig
-RUN wget -q -O - https://www.freedesktop.org/software/fontconfig/release/fontconfig-2.12.6.tar.bz2 | \
-    tar jxvf -
+RUN wget -q -O - https://www.freedesktop.org/software/fontconfig/release/fontconfig-${FONTCONFIG_VERSION}.tar.gz | \
+    tar zxvf -
 WORKDIR fontconfig-${FONTCONFIG_VERSION}
 RUN ./configure --prefix=${C} --host=${CROSS_ARCH} --build=${BUILD_ARCH} --enable-static --disable-shared && \
     make && \
@@ -521,5 +520,4 @@ RUN $CROSS_ARCH-strip bin/*.exe
 
 # Create mingw64 archive
 RUN zip -r -9 audiowaveform-mingw64.zip bin/ etc/ include/ lib/ share/
-
 

